hack/update-codegen.sh

```diff
@@ -248,12 +248,27 @@ gen-conversions() {
   clean pkg/internal/apis 'zz_generated.conversion.go'
   clean pkg/webhook/handlers/testdata/apis 'zz_generated.conversion.go'
   echo "Generating conversion functions..." >&2
-  prefixed_inputs=( "${conversion_inputs[@]/#/$module_name/}" )
-  joined=$( IFS=$','; echo "${prefixed_inputs[*]}" )
-  "$conversiongen" \
-    --go-header-file hack/boilerplate/boilerplate.generatego.txt \
-    --input-dirs "$joined" \
+  # meta api
+  "$conversiongen" --go-header-file hack/boilerplate/boilerplate.generatego.txt \
+    --input-dirs github.com/jetstack/cert-manager/pkg/internal/apis/meta/v1 \
+    -O zz_generated.conversion
+  # acme and certmanager apis
+  for v in v1alpha2 v1alpha3 v1beta1 v1
```

Comment:
How come we're now looping through for each API version rather than having the generation done as before? This seems odd 🤔

Reply:
PTAL at this first https://github.com/kubernetes/kubernetes/issues/101567 .

What I can tell from my debugging the `conversion-gen` behavior has changed quite a bit since 1.18.x release. So, we need to generate the conversion files in order (meaning acme/$v first, then certmanager/$v, since certmanager/$v used the generated files from acme/$v). So, this seemed like a reasonable way to do so.